app/models/project_media.rb

```diff
@@ -69,4 +74,50 @@ def add_elasticsearch_data
     #ElasticSearchWorker.perform_in(1.second, YAML::dump(ms), YAML::dump({}), 'add_parent')
   end
 
+  def cached_annotations(type = nil)
```

Comment:
These annotations are no longer cached, and we should rename the method accordingly.